data/hai missions.txt

```diff
@@ -1061,3 +1061,93 @@ mission "Hai Honeymoon"
 	on complete
 		payment 100000
 		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`
+
+
+
+mission "Enlisted by Hai [1]"
+	name "Defend <planet>"
+	description "Defeat a pirate raid on <destination>."
+	to offer
+		"combat rating" > 400
+		"random" < 35
+	source "Allhome"
+	npc
+		government "Hai"
+		personality "heroic" staying"
+		fleet "Large Hai" 2
+		fleet "Large Militia"
+	npc evade
+		government "Pirate"
+		personality "heroic" "staying"
+		fleet "Large Northern Pirates" 10
+	on offer
+		conversation
+			`Suddenly you notice a massive rush in the spaceport: "We are under attack! <planet> is under attack by the unfettered humans! We need every combat-worthy ship to join in the defenses!" The authorities will probably be very gracious, but this could also be an easy way to get yourself killed.`
+			choice
+				`	(Join the defense fleet.)`
+				`	(Stay here until the fight is over.)`
+					defer
+			`A mix of Hai and Human pilots have amassed to help repel the unfettered attack. You join them, and take off together...`
+				launch
+	on complete
+		dialog
+			`The government of <planet> pays you <payment> for helping to drive off the attack.`
+		payment 280000
+
+mission "Enlisted by Hai [2]"
+	name "Enlist with the Hai"
+	description "You have been recommended for your attentiveness for keeping the peace throughout Hai space. Hai officials would like to consult you about possible enlistment."
```

Comment:
Is it really "attentiveness" if the player only does a single defense mission?

Reply:
Yeah..  never really digged using `attentiveness` there anyway, could change `You have been recommended for your attentiveness for keeping the peace throughout Hai space.` -> `You have been recommended following your demonstration of peace.`